api/api.py

```diff
@@ -42,7 +42,7 @@ def get_co2_estimate(
     else:
         return {
             "error": "Not enough information was provided to calculate CO2 estimate."
-        }
+        }, 400
```

Comment:
Even in Python it's best avoided to have different return types depending on your input etc. Or does this style come from connexion?

Reply:
Maybe its better to wrap in some Response class? 
Indeed it is from Flask docs https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses

```python
@app.errorhandler(404)
def not_found(error):
    return render_template('error.html'), 404
```

Reply:
Always return result, status_code tuple